master/buildbot/test/unit/test_reporters_irc.py

```diff
@@ -53,6 +53,14 @@ def makeBot(self, *args, **kwargs):
             args = ('nick', 'pass', ['#ch'], [], [], {})
         return irc.IrcStatusBot(*args, **kwargs)
 
+    def test_groupDescribe(self):
+        b = self.makeBot()
+        b.describe = lambda d, m: evts.append(('n', d, m))
```

Comment:
what `d` and `m` are?

Reply:
`d` is for destination such as an IRC user or channel, `m` is for message.
It's not the best variable names, but I don't think it is fair to penalize @zware for this, because this convention is followed all over the place in this file.